network/wsNetwork.go

```diff
@@ -1951,7 +1914,7 @@ const ProtocolVersionHeader = "X-Algorand-Version"
 const ProtocolAcceptVersionHeader = "X-Algorand-Accept-Version"
 
 // SupportedProtocolVersions contains the list of supported protocol versions by this node ( in order of preference ).
-var SupportedProtocolVersions = []string{"2.2", "2.1"}
+var SupportedProtocolVersions = []string{"2.2"}
```

Comment:
The idea being any nodes that only supported 2.1 should be offline since several consensus upgrades have passed since then.